types/defines/rpc.d.ts

```diff
@@ -195,22 +195,35 @@ declare module "cloudflare:workers" {
     webSocketError?(ws: WebSocket, error: unknown): void | Promise<void>;
   }
 
-  export type DurationLabel =
+  export type WorkflowDurationLabel =
     | "second"
     | "minute"
     | "hour"
     | "day"
     | "week"
     | "month"
     | "year";
-  export type SleepDuration = `${number} ${DurationLabel}${"s" | ""}` | number;
 
-  type WorkflowStep = {
+  export type WorkflowSleepDuration = `${number} ${WorkflowDurationLabel}${"s" | ""}` | number;
+
+  export type WorkflowBackoff = 'constant' | 'linear' | 'exponential';
+
+  export type WorkflowStepConfig = {
+    retries?: {
+      limit: number;
+      delay: string | number;
+      backoff?: WorkflowBackoff;
+    };
+    timeout?: string | number;
+  };
+
```

Comment:
I realise there are already Workflow types in this file, but I think they should probably be in a separate `workflows.d.ts` so we can more easily separate them.